src/feature_lib.cpp

```diff
@@ -0,0 +1,21 @@
+#include "feature_lib.h"
+#include "conversions.h"
+
+namespace featurelib {
+
+SiftDetector::SiftDetector() {}
+
+featurelib::DetectionResult SiftDetector::detect(const std::vector<std::uint8_t> &input_vector) {
```

Comment:
Let's call the parameter "input" -- no need to repeat its type in its name

Reply:
done